Features/ModerationCommands/BanCommand.cs

```diff
@@ -75,8 +68,26 @@ await _databaseService.NewBanRecordAsync(
                 Timestamp = DateTime.UtcNow
             };
 
+            var dmEmbed = new DiscordEmbedBuilder
+            {
+                Title = "You Have Been Banned",
+                Color = DiscordColor.Red,
+                Description = $"**Server:** {ctx.Guild.Name}\n" +
+                            $"**Reason:** {reason}\n" +
```

Comment:
If the reason is `null`, don't show "**Reason:** " but instead say "No reason provided."